meta-resin-common/recipes-support/resin-init/resin-init-flasher/resin-init-flasher

```diff
@@ -39,13 +39,14 @@ INTERNAL_DEVICE_CONF_PART_MOUNTPOINT=/tmp/internal_conf
 FLASHER_CONF_FILE=/etc/resin-init-flasher.conf
 
 function prepare_uboot {
-    UBOOT_FROM_EMMC="mmcdev=$INTERNAL_DEVICE_UBOOT\n"
-    UBOOT_FROM_EMMC=$UBOOT_FROM_EMMC"bootpart=$INTERNAL_DEVICE_UBOOT:2\n"
-    # Second partition hardcoded!!!
-
-    if ! echo -e ${UBOOT_FROM_EMMC} > $INTERNAL_DEVICE_BOOT_PART_MOUNTPOINT/uEnv.txt; then
+    if ! mv $EXTERNAL_DEVICE_BOOT_PART_MOUNTPOINT/uEnv.txt $INTERNAL_DEVICE_BOOT_PART_MOUNTPOINT ; then
```

Comment:
Why mv and not cp? If you move you might break reflashing with the same flasher device.